api/routes/user.js

```diff
@@ -0,0 +1,60 @@
+const User = require('../models/User');
+const express = require('express');
+const { auth } = require('../middleware');
+
+const app = express();
+
+const updateUser = (req, res) => {
+    const { id } = req.params;
+
+    if (req.user._id != id) {
```

Comment:
@abelRoland you only want to allow updating your own user account. If you don't add this anyone on the internet can update the details of the user including password.